mypy_django_plugin/transformers/managers.py

```diff
@@ -115,6 +128,14 @@ def get_funcdef_type(definition: Union[FuncBase, Decorator, None]) -> Optional[P
     )
 
 
+def _get_funcdef_type(definition: Union[FuncBase, Decorator, None]) -> Optional[ProperType]:
```

Comment:
We could do `mypy.nodes.Node` here I think.

```suggestion
def _get_funcdef_type(definition: Union[Node, None]) -> Optional[ProperType]:
```